GraphicsAPI_Test/main.cpp

```diff
@@ -255,7 +257,11 @@ int main() {
 
     // Swapchain
     const uint32_t swapchainCount = 3;
+#ifdef XR_TUTORIAL_USE_VULKAN    
     swapchainFormat = apiType == VULKAN ? VK_FORMAT_B8G8R8A8_UNORM : DXGI_FORMAT_B8G8R8A8_UNORM;
+#else
+    swapchainFormat = DXGI_FORMAT_B8G8R8A8_UNORM;
+#endif
```

Comment:
Everything else looks fine but this and the above change looks unrelated? or was it just the change required to actually build?

Reply:
that was required to make build work on a machine without vulkan